helios-services/src/main/java/com/spotify/helios/agent/Supervisor.java

```diff
@@ -34,12 +40,6 @@
 import java.io.InterruptedIOException;
 import java.util.concurrent.TimeUnit;
 
-import static com.google.common.base.Preconditions.checkNotNull;
-import static com.google.common.util.concurrent.MoreExecutors.directExecutor;
-import static com.spotify.helios.common.descriptors.TaskStatus.State.STOPPED;
-import static com.spotify.helios.common.descriptors.TaskStatus.State.STOPPING;
-import static java.util.concurrent.TimeUnit.SECONDS;
-
 /**
```

Comment:
@saunakchakrabarti Yes, albeit automatically. My IntelliJ should be configured with the Spotify style guide, and does automatic import sorting so that I don't go crazy fighting Checkstyle.